bigtable/autoscaler/strategies.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2017 Google Inc.
```

Comment:
I see. If that's valuable and matches with how this will be presented in the docs, then that's fine.